dotnetv3/Bedrock-runtime/Models/CohereCommand/Command_R_InvokeModelWithResponseStream/InvokeModelWithResponseStream.csx

```diff
@@ -0,0 +1,59 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// snippet-start:[BedrockRuntime.dotnetv3.InvokeModelWithResponseStream_CohereCommandR]
+// Use the native inference API to send a text message to Cohere Command R
+// and print the response stream.
+
+using Amazon;
+using Amazon.BedrockRuntime;
+using Amazon.BedrockRuntime.Model;
+using System;
+using System.IO;
+using System.Text.Json;
+using System.Text.Json.Nodes;
+
+// Create a Bedrock Runtime client in the AWS Region you want to use.
+var client = new AmazonBedrockRuntimeClient(RegionEndpoint.USEast1);
+
+// Set the model ID, e.g., Command R.
+var modelId = "cohere.command-r-v1:0";
+
+// Define the user message.
+var userMessage = "Describe the purpose of a 'hello world' program in one line.";
+
+//Format the request payload using the model's native structure.
+var nativeRequest = JsonSerializer.Serialize(new
+{
+    message = userMessage,
+    max_tokens = 512,
+    temperature = 0.5
+});
+
+// Create a request with the model ID, the user message, and an inference configuration.
+var request = new InvokeModelWithResponseStreamRequest()
+{
+    ModelId = modelId,
```

Comment:
Where is the inference configuration? I only see the model and the body.

Reply:
The inference configuration is defined in the request object above. But, yes, the comment is confusing. I'll clarify.